js/docker/demo.dockerfile

```diff
@@ -22,9 +22,19 @@ ENTRYPOINT ["yarn", "start:dev"]
 
 FROM lerna-bootstrap as application-front-end
 
+RUN yarn global add serve
```

Comment:
can't be worse than the dev server, I suspect.  I have usually seen this one used https://www.npmjs.com/package/http-server  But seems like the same deal :)